drivers/SmartThings/zwave-sensor/src/test/test_ezmultipli_multipurpose_sensor.lua

```diff
@@ -43,6 +49,126 @@ end
 
 test.set_test_init_function(test_init)
 
+test.register_message_test(
+    "Basic report (contact sensor) should be handled",
+    {
+      {
+        channel = "zwave",
+        direction = "receive",
+        message = { mock_device.id, zw_test_utils.zwave_test_build_receive_command(Basic:Report({ value = 0xFF })) }
+      },
+      {
+        channel = "capability",
+        direction = "send",
+        message = mock_device:generate_test_message("main", capabilities.switch.switch.on())
+      }
+    }
+)
+
+test.register_message_test(
+    "Basic report (contact sensor) should be handled",
```

Comment:
can we add some more description to the test case to differentiate from the 1st test case ?

Reply:
fixed!